### ISSUE TITLE
Don't close term on Save

### ISSUE THREAD

**Issue by jamesdeluk** [jzohrab/lute-v3#217]
Currently when you click Save in the top-right box, the box empties after a "saved" message.

Often, after creating a new term, I need to access it again (to add a parent to the parent), meaning I have to reclick the original term to reopen in.

I would prefer it if, after clicking Save, there was a "saved" message but the term stayed in the box.

It would also make it easier if you make a mistake and need to edit (e.g. a typo).

**Comment by jzohrab** [comment]
Right, that makes sense, thanks.  I closed it to remove visual clutter while reading, but perhaps leaving it open is just fine.

**Comment by jzohrab** [comment]
Will post this in ideas forum later to see if anyone is vehemently opposed to the change.  It shouldn't matter, and I don't think would influence my decision, just curious.  Users can still hit "escape" to get back to hover mode which closes the form.
